Resources/Prototypes/Entities/Objects/Magic/books.yml

```diff
@@ -20,7 +20,7 @@
   components:
     - type: Spellbook
       spells:
-        FlashRune: -1
+        ActionFlashRune: -1
```

Comment:
```suggestion
```

This can be removed since I didn't realize someone made a scroll with runes in it.